mlflow/tracking/fluent.py

```diff
@@ -140,6 +140,11 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
                 "Cannot start run with ID {} because it is in the "
                 "deleted state.".format(existing_run_id)
             )
+        end_time = active_run_obj.info.end_time
+        MlflowClient().get_store().update_run_info(
+            existing_run_id, run_status=RunStatus.RUNNING, end_time=end_time
```

Comment:
Hmmm...I'm a bit confused by this change. Looks like we're preserving the same `end_time` that was initially set when the run was ended the first time, even though the run was resumed later.

This gets overwritten when the resumed run ends again, right? So maybe that's okay. Ideally, it'd be nice if we could somehow unset this field; does `update_run_info` support that?

Reply:
Discussed offline